hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcachedMultiendpointTest.java

```diff
@@ -20,15 +20,8 @@
 import com.hazelcast.config.Config;
 import com.hazelcast.config.JoinConfig;
 import com.hazelcast.config.ServerSocketEndpointConfig;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.QuickTest;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
 
-@RunWith(HazelcastParallelClassRunner.class)
-@Category(QuickTest.class)
-public class MemcachedMultiendpointTest
-        extends MemcachedTest {
+public class MemcachedMultiendpointTest extends MemcachedTest {
```

Comment:
MemcachedTest is 

```
@RunWith(HazelcastParallelClassRunner.class)
@Category({QuickTest.class, ParallelJVMTest.class})
```

fyi if you want to remove parallel jvm test from this one as well.

Reply:
Yes, I think this one is safe to be ParallelJVMTest, the parent is and we only use slightly different instance config.